src/lib/escape-highlight.ts

```diff
@@ -1,5 +1,6 @@
+import escape from '../lib/utils/escape';
```

Comment:
instead escape-highlight should be put inside `utils` and the import from `./escape` I think (then all references to escape-highlight need to be moved too)

Reply:
Everything should be fixed in https://github.com/algolia/instantsearch.js/pull/4618/commits/313d37f0f6540100857f884b395121e0dd2d73cf

Reply:
False alarm, some files didn't throw errors during tests

Reply:
Now fixed in https://github.com/algolia/instantsearch.js/pull/4618/commits/25110df6511569ee41bc6bae0d25087a68786efa